addons/controllers/csi/vspherecsiconfig_utils.go

```diff
@@ -326,3 +332,49 @@ func (r *VSphereCSIConfigReconciler) overrideMiscValues(dvscsi *DataValuesVSpher
 		dvscsi.InsecureFlag = *config.InsecureFlag
 	}
 }
+
+func (r *VSphereCSIConfigReconciler) constrainNumberOfDeploymentReplicas(ctx context.Context, proposedCount int) int {
```

Comment:
Suggestion: wondering if things would be easier to do `proposedCount int32` return `int32` as well? So that caller does not have to do several conversions.

Reply:
updated